### ISSUE TITLE
Compile error with --advance 5

### ISSUE THREAD

**Issue by usmanov-danil** [dashingsoft/pyarmor#760]
1. On MacOS 12.3.1 run pyarmor to obfuscate the script inside the docker container, python 3.7, PyArmor 7.5.0
```
$ PYTHONOPTIMIZE=2 pyarmor obfuscate --advanced 5 --exclude core/internal/code/api/streamer.py --src="." -r --output=/new_app entry.py 
#22 37.29 INFO     Generate module spp code successfully
#22 37.29 INFO     Compiling spp code ...
#22 37.40 INFO     Compile spp code successfully
#22 37.47 INFO          core/internal/code/pipeline/pipeline.py -> ../new_app/core/internal/code/pipeline/pipeline.py
#22 37.48 INFO     Generating spp code for <frozen core.internal.code.pipeline.pipeline> ...
#22 37.51 ERROR    'Set' object has no attribute 'ctx'
```

And if I add `# pyarmor options: no-spp-mode` comment another error occurred in the other files:
```
PYTHONOPTIMIZE=2 pyarmor obfuscate --advanced 5 --exclude core/internal/code/api/streamer.py --src="." -r --output=/new_app entry.py 

#22 51.86 INFO     Compiling spp code ...
#22 51.87 /tmp/pyarmor_frozen core.internal.tools.metrics.trackeval.metrics.track_map.c:395:25: error: redefinition of ‘_listcomp_326_1’
#22 51.87  static inline PyObject *_listcomp_326_1 (PyObject **ctx)
#22 51.87                          ^~~~~~~~~~~~~~~
#22 51.87 /tmp/pyarmor_frozen core.internal.tools.metrics.trackeval.metrics.track_map.c:355:25: note: previous definition of ‘_listcomp_326_1’ was here
#22 51.87  static inline PyObject *_listcomp_326_1 (PyObject **ctx)
#22 51.87                          ^~~~~~~~~~~~~~~
#22 51.89 ERROR    Command '['gcc', '-O3', '-c', '-fPIC', '-fno-asynchronous-unwind-tables', '-fno-unwind-tables', '-o', '/tmp/pyarmor_frozen core.internal.tools.metrics.trackeval.metrics.track_map.o', '/tmp/pyarmor_frozen core.internal.tools.metrics.trackeval.metrics.track_map.c']' returned non-zero exit status 1.
```

Such error appears in files where I use numpy==1.21.6 and opencv==4.5.3

**Comment by jondy** [comment]
Could you send `core/internal/code/pipeline/pipeline.py` and `core/internal/tools/metrics/trackeval/metrics/track_map.py` to pyarmor@163.com?

**Comment by jondy** [comment]
I have find the problem, you need not send me the scripts. It will be fixed soon.

**Comment by jondy** [comment]
All of these bugs (include #758) has been fixed in the pre-relase v7.5.1, it can be tested by this temporary link

      pip install https://pyarmor.dashingsoft.com/downloads/temp/pyarmor-7.5.1.zip

**Comment by jondy** [comment]
Fixed in v7.5.1
